code/src/java/pcgen/gui2/converter/DefaultTokenProcessor.java

```diff
@@ -31,42 +30,35 @@
 	@Override
 	public String process(TokenProcessEvent tpe)
 	{
-		try
```

Comment:
Similar to above - previous design is fail of a single token is just that - now it will bubble up to... where?

Reply:
it won't bubble up, it cannot possibly be thrown